### DISCUSSION TITLE
Could PiSlide support a dedicated portrait (rotated screen) mode?

### DISCUSSION THREAD

**Discussion by freddieFishCake** [JarvyJ/pislide-os#20] [Feature Suggestions]
Portrait pictures don't display well on a landscape monitor. Supporting a 90 degree rotated screen would be nice for collections of photographs that are in portrait mode.

**Comment by JarvyJ** [comment]
Yes, we have plans to support screen rotations in https://github.com/JarvyJ/pislide-os/issues/18 !
